components/card.tsx

```diff
@@ -0,0 +1,39 @@
+import Image from 'next/image';
+import ReactMarkdown from 'react-markdown'
+import { Remarkable } from 'remarkable'
+import IEvent from '../utils/IEvent';
+import { GithubSlugger } from 'github-slugger-typescript'
+
+const WorkshopCard = (props: IEvent) => {
+  const md = new Remarkable();
+  const parsedDescription = md.render(props.description);
+  const date = new Date(props.date)
+  const slugger = new GithubSlugger();
+  const eventUrl = "https://events.purduehackers.com/" + slugger.slug(props.name)
+
+  return(
+    <div className="max-w-sm overflow-hidden bg-white rounded-lg shadow-lg">
+    <div className="px-6 py-4">
+      <a className="mb-2 text-xl font-bold underline text-declaration hover:text-amber-500" href={eventUrl} target="_blank">{props.name}</a>
+      <p className='text-slate-500'>{date.toDateString()}</p>
+      <p className="h-40 overflow-auto text-base text-gray-700">
+        <div dangerouslySetInnerHTML={{__html:parsedDescription}}></div>
+      </p>
+    </div>
+    <div className='relative w-full h-52'>
+      <Image
+        src={props.img}
+        layout='fill'
+        objectFit='contain'
+      ></Image>
+    </div>
+    
+    <div className="px-6 pt-4 pb-2">
+      <span className="inline-block px-3 py-1 mb-2 mr-2 text-sm font-semibold text-gray-700 bg-gray-200 rounded-full">{props.location}</span>
+      <span className="inline-block px-3 py-1 mb-2 mr-2 text-sm font-semibold text-gray-700 bg-gray-200 rounded-full">RSVP: {props.rsvp}</span>
```

Comment:
I think `RSVP` as a stat is on the right track, but I think people will view that as how many people showed up, and RSVP is almost never a good representation of that.

A slightly more complicated but I think better way to do it is to check if the record has a stat field (either `Stat 1 Label`, `Stat 2 Label`, or `Stat 3 Label`) that === `people`, and then getting the number from `Stat {num} Data`. If there's no `people`, then hide the component altogether.

<img width="294" alt="Screenshot 2022-11-29 at 12 01 27 PM" src="https://user-images.githubusercontent.com/14811170/204594421-f9d66e10-08c3-4df1-9d36-493ed0cf2f85.png">

Unfortunately I didn't build the recap feature with this use case in mind, so it'll probably be a little hacky to implement, but I think it's worth it.